### DISCUSSION TITLE
diffCapturePoint

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#22] [Q&A] [answered]
AutoBalancerのInPort `diffCapturePoint`は使われていない、という認識であってますか。何か使うデモがありますか?

**Comment by YutaKojio** [comment, marked as answer]
はい，今は使ってないです．
AutoBalancer rtcで予見制御で着地位置修正，Stabilizer rtcで状態推定してたときの名残で
現在は結局auto-stabilizerでは予見制御自体動かなくなってるんじゃないかと思います
